api/src/env.ts

```diff
@@ -204,6 +204,9 @@ const allowedEnvironmentVars = [
 	'ADMIN_PASSWORD',
 	// telemetry
 	'TELEMETRY',
+	'TELEMETRY_INGRESS',
```

Comment:
Just curious: Was there any reason for not calling this env var `TELEMETRY_URL`? 🤔

Reply:
I ehh.. it eh.. sounds more smart?

Good shout though, lets keep it consistent 🙂